src/main/java/com/github/cchacin/cucumber/steps/DatabaseSteps.java

```diff
@@ -95,4 +105,37 @@ public void I_have_the_following_sql_script(final String script) throws Throwabl
         new ScriptRunner(this.destination.getConnection())
                 .runScript(new BufferedReader(new FileReader(Thread.currentThread().getContextClassLoader().getResource(script).getPath())));
     }
+
+    @Then("^I should have the following rows in the \"([^\"]*)\" table:$")
+    public void I_have_the_following_rows_in_the_table(
+        final String tableName, final DataTable data) throws SQLException, ClassNotFoundException {
+        exists(tableName, data);
+    }
+
+    void exists(final String tableName, final DataTable data) throws SQLException, ClassNotFoundException {
+        final List<DataTableRow> rows = data.getGherkinRows();
+        final List<String> columns = rows.get(0).getCells();
+        StringBuilder queryBuilder = new StringBuilder();
+        queryBuilder.append(String.format("SELECT * FROM %s WHERE ", tableName));
+        queryBuilder.append(StringUtils.join(columns, " = ? AND "));
+        queryBuilder.append(" = ?;");
+        try (Connection conn = getConnection()) {
+            PreparedStatement stmt = conn.prepareStatement(queryBuilder.toString());
+            for (DataTableRow row : rows.subList(1, rows.size())) {
+                List<String> rowValues = row.getCells();
+                for (int i = 0; i < columns.size(); i++) {
+                    stmt.setString(i + 1, rowValues.get(i));
+                }
+                ResultSet rs = stmt.executeQuery();
```

Comment:
fixed in https://github.com/xsuoownit/cucumber-common-steps/commit/2ee35d2b82cfaa5f0a4395abd508e07301ec318e